Rajinderpal Singh's Final Report for DSC-X (OMG-Frames)

rajinderpalsingh2001 opened the pull request:
My Final Report for Winter of Code 2020 for the Project OMG-Frames under the Organization DSC-X.

rajinderpalsingh2001 committed:
Rajinderpal Singh's Final Report for DSC-X (OMG-Frames)

My Final Report for Winter of Code 2020 for the Project OMG-Frames under the Organization DSC-X.

xprilion commented on DSC-X/rajinderpal_singh_WoC.md:
```diff
@@ -0,0 +1,114 @@
+OMG-Frames aims to Generate Badges for the events that happens for across the Student Clubs, or any other events.My work is to create a Frontend for the entire project mainly Dashboard.
+>**Apart from my work, I added new features which I thought to be in for better user experience.**
+
+**I am <a href="https://github.com/sairish2001/" target="_blank">*Rajinderpal Singh*</a>, an undergraduate student from <a href="http://www.gndu.ac.in/" target="_blank">*Guru Nanak Dev University,Amritsar*</a>, as part of Winter of Code.** This project was mentored by <a href="https://github.com/xprilion/" target="_blank">Anubhav Singh</a>
```
Convert html to markdown

xprilion commented on DSC-X/rajinderpal_singh_WoC.md:
```diff
@@ -0,0 +1,114 @@
+## Contributions
+Contributions are outlined as follows:
+
+ * 20 <a href="https://github.com/dsc-x/omg-frames/commits/develop" target="_blank">commits</a> (approx.) are made into the develop branch of iWasAt.events
```
use markdown

xprilion reviewed (changes requested):
Please correct the filename

Also convert HTML to markdown unless absolutely necessary. This allows us to deploy the site with consistent styling

rajinderpalsingh2001 committed:
Delete rajinderpal_singh_WoC.md

rajinderpalsingh2001 committed:
Report Updated! Follows markdown's convention

My Final Report for Winter of Code 2020 for the Project OMG-Frames under the Organization DSC-X.

rajinderpalsingh2001 committed:
Changed * from - for unordered list view

xprilion reviewed (changes requested):
@sairish2001 please follow the filename guidelines on the project readme.
